Source/Core/DolphinWX/Frame.cpp

```diff
@@ -359,14 +359,14 @@ bool CFrame::InitControllers()
 #if defined(HAVE_X11) && HAVE_X11
 		Window win = X11Utils::XWindowFromHandle(GetHandle());
 		HotkeyManagerEmu::Initialize(reinterpret_cast<void*>(win));
-		Pad::Initialize(reinterpret_cast<void*>(win));
-		Keyboard::Initialize(reinterpret_cast<void*>(win));
-		Wiimote::Initialize(reinterpret_cast<void*>(win));
+		Pad::Initialize             (reinterpret_cast<void*>(win));
+		Keyboard::Initialize        (reinterpret_cast<void*>(win));
+		Wiimote::Initialize         (reinterpret_cast<void*>(win));
```

Comment:
please don't do this sort of thing with functions, leave the actual argument lists against the call.